### PR TITLE
Add some certification tests to CI

### PR DESCRIPTION
**Pull request by raju-apple**
#### Problem
Adding following Yaml files(certification tests) : 
	src/app/tests/suites/certification/Test_TC_CC_1_1.yaml (Color Cluster Test case,  ref: 27.1.1. [TC-CC-1.1] Global attributes with server as DUT)
	src/app/tests/suites/certification/Test_TC_LVL_1_1.yaml( Level Control Cluster Test case, ref :24.1.1. [TC-LVL-1.1] Global attributes with server as DUT)
	src/app/tests/suites/certification/Test_TC_MC_1_1.yaml (Media Cluster Library Test Case , ref 21.1.1. [TC-MC-1.1] Global attributes with server as DUT)
	src/app/tests/suites/certification/Test_TC_PCC_1_1.yaml (Pump Control Cluster Test Case , ref 15.1.1. [TC-PCC-1.1] Global attributes with server as DUT)
	src/app/tests/suites/certification/Test_TC_RH_1_1.yaml (Relative Humidity Measurement Cluster , ref 9.1.1. [TC-RH-1.1] Global attributes with server as DUT)
	src/app/tests/suites/certification/Test_TC_TSTAT_1_1.yaml (Thermostat Cluster test case , ref 
42.1.1. [TC-TSTAT-1.1] Global attributes with server as DUT)
	src/app/tests/suites/certification/Test_TC_TSUIC_1_1.yaml ( Thermostat user configuration test case , ref 12.1.1. [TC-TSUIC-1.1] Global attributes with server as DUT)

#### Testing
How was this tested? (at least one bullet point required)
* Ran the tests locally , some of the default values for ClusterRevision do not match the spec , have marked them as disabled for now till it's resolved.

### CODE REVIEW

**Comment by andy31415** [comment]
Could you add some description that does not sound as cryptic?

Without looking at the code, `cc lvl mc pcc rh tstat ts UI c 1 1` tells me nothing nor do the yaml file names help me

**Review by andy31415 — DISMISSED** [review]
Request changes: in the PR description explain what cluster tests (in english, no shortcut like cc for color cluster or tstat for thermostat) are added.

Will make it easy for anyone looking at the log to understand what goes in.

**Comment by raju-apple** [comment]
> Request changes: in the PR description explain what cluster tests (in english, no shortcut like cc for color cluster or tstat for thermostat) are added.
> 
> Will make it easy for anyone looking at the log to understand what goes in.

Done , added more detail in the description.

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@project-chip/reviewers-google @project-chip/reviewers-samsung? @project-chip/reviewers-nordic ?

**Comment by woody-apple** [comment]
Note, I opened up this project: https://github.com/project-chip/connectedhomeip/projects/75
@raju-apple filed spec compliance issues there. More coming

**Review by jmartinez-silabs — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_CC_1_1.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      disabled: true
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 4
+
+    - label:
+          "write the default values to mandatory global attribute:
+          ClusterRevision"
+      disabled: true
```
Comment (vivien-apple) [inline src/app/tests/suites/certification/Test_TC_CC_1_1.yaml:32]: You can remove disabled for all `writeAttribute` action now.

**Review by vivien-apple — APPROVED** [review]
If I have read the tests correctly, most `ClusterRevision` are wrong in the xml definition of `src/app/zap-templates/zcl/` :'(

Approved assuming you will remove the disabled flag for all `writeAttribute` since those are supported now.

**Review by andy31415 — APPROVED** [review]

File: examples/chip-tool/templates/tests-commands.zapt [hunk 2]
```diff
@@ -4,5 +4,5 @@
...
+{{>test_cluster tests="TV_TargetNavigatorCluster, TV_AudioOutputCluster, TV_ApplicationLauncherCluster, TV_KeypadInputCluster, TV_AccountLoginCluster, TV_WakeOnLanCluster, TV_ApplicationBasicCluster, TV_MediaPlaybackCluster, TV_TvChannelCluster, TV_LowPowerCluster, TV_MediaInputCluster, TestCluster, TestConstraints, TestDelayCommands, TestSubscribe_OnOff, Test_TC_OO_1_1, Test_TC_OO_2_1, Test_TC_OO_2_2, Test_TC_DM_1_1, Test_TC_DM_3_1, Test_TC_CC_3_4, Test_TC_CC_5, Test_TC_CC_6, Test_TC_CC_7, Test_TC_CC_8, Test_TC_WNCV_1_1, Test_TC_WNCV_2_1, Test_TC_BI_1_1, Test_TC_FLW_1_1, Test_TC_TM_1_1, Test_TC_OCC_1_1, TestOperationalCredentialsCluster, Test_TC_LVL_1_1, Test_TC_CC_1_1, Test_TC_RH_1_1, Test_TC_MC_1_1, Test_TC_TSTAT_1_1, Test_TC_PCC_1_1, Test_TC_TSUIC_1_1"}}
```
Comment (andy31415) [inline examples/chip-tool/templates/tests-commands.zapt:7]: Do we have some readme that explains the test naming format? Like what does TC/CC/LVL/PCC and such mean and what do the numerics mean (most are 1_1, but I see 2_1, 2_2 and so on ... what are they?)

Comment (woody-apple) [reply examples/chip-tool/templates/tests-commands.zapt:7]: This is how the certification team is formatting their test plans.

Comment (raju-apple) [reply examples/chip-tool/templates/tests-commands.zapt:7]: Yeah that's the cert test cases language Ex : 15.1.1. [TC-PCC-1.1] Global attributes with server as DUT

File: src/app/tests/suites/certification/Test_TC_CC_1_1.yaml (hunk 1 above)
Comment (raju-apple) [reply src/app/tests/suites/certification/Test_TC_CC_1_1.yaml:32]: Vivien , in all the 20 odd yamls that we have the write attributes are disabled for now . Let me verify each of them and put up a separate PR to enable them all .

**Pull request merged**